README.md

```diff
@@ -2,7 +2,7 @@ Hello from CUHK CSCI3251! This is another playground for our course students to
 
 There are some teams in our course project:
 
-* [Team X](https://csci3251-2020.github.io/project-team-x/ 'Team-X') _@chuckjee_ (Please remove this line when you add the first actual team below)
+* [Team J](https://csci3251-2020.github.io/project-team-j/ 'Team-J')
```

Comment:
Almost there! Please sign your username like in the demo line.

Reply:
Done!